osbuild/tmpfs.py

```diff
@@ -27,10 +27,9 @@ def __enter__(self):
         return self.root
 
     def __exit__(self, exc_type, exc_value, exc_tb):
-        if not self.root:
-            return
-        if self.mounted:
-            subprocess.run(["umount", "--lazy", self.root], check=True)
-            self.mounted = False
-        os.rmdir(self.root)
-        self.root = None
+        if self.root:
```

Comment:
I personally very much prefer the golang-like early return style to nest ifs. I agree with the reasoning that this is easier to follow (and pythons is so much indented anyway). Unless others really strongly feel the other way. There os also nothing else changed here, right, so this is kinda unrelated?

Reply:
That was to unify the `return`, since my initial version had `return None`. I only then realized that `return` is equal to `return None` in python.

So yeah, fine with me.